packages/graphback-core/tests/MetadataTest.ts

```diff
@@ -0,0 +1,33 @@
+//tslint:disable-next-line: match-default-export-name no-implicit-dependencies
+import { readFileSync } from 'fs';
+import { buildSchema } from 'graphql';
+import { GraphbackCoreMetadata } from '../src'
+import { existedOperationTypeMessage } from 'graphql/validation/rules/UniqueOperationTypes';
+
+const schemaText = readFileSync(`${__dirname}/mock.graphql`, 'utf8')
```

Comment:
I'm personally not a fan of loading the mock data model from a separate file because it is difficult to quickly see what the model looks like from the test when updating or reviewing. Because this file that has not changed it is not included in the PR.

Including the schema in the test makes it easier to read

https://github.com/aerogear/graphback/blob/dbc498831d241913fe9262449e1ae93a00045a1a/packages/graphback-core/tests/RelationshipMetadataBuilderTest.ts#L69

Reply:
There are multiple instances of that so I will create issue to get that done